### DISCUSSION TITLE
Display TTGO battery voltage on Webpages

### DISCUSSION THREAD

**Discussion by Medvidas** [dl9rdz/rdz_ttgo_sonde#399] [Q&A] [answered]
Hi,
I am wondering if it is there any way how to display Rdzsonde battery voltage on TTGO Web pages?

The point is that I am running remote TTGO with battery and small solar panel, 
and it needs sometimes to be externally recharged if no sunlight for couple of days.
Voltage can be show on local OLED, but should be nice to see it remotely...

Thanks

**Comment by dl9rdz** [comment, marked as answer]
I have some plans on adding more information on the web interface, like communication status (aprs, sondehub) and also I can add battery information. So it is on the TODO list (maybe by end of the year)

**Comment by Medvidas** [comment]
Great, I'm looking forward to the new features. That update will be possible nice Christmas present for all users.
Regards

**Comment by Medvidas** [comment]
Works flawlessly in the last dev versions, thanks
